### DISCUSSION TITLE
Melihat version panda saja error, Help !

### DISCUSSION THREAD

**Discussion by imronovsky2000** [teamalgoritma/community#868] [Q&A] [answered]
Hallo , ini pertanyaan pertama saya di GitHub.
Mohon bantuannya untuk error ini kenapa ya?
```
import pandas as pd
import numpy as np
import matplotlib as plt

```
command :
```
pd.show_versions()
```
Error:
```
TypeError                                 Traceback (most recent call last)
Cell In[21], line 1
----> 1 pd.show_versions()

TypeError: 'str' object is not callable

```

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Pak @imronovsky2000 ,
Jika Anda ingin melihat versi Pandas yang digunakan, Anda bisa menggunakan sintaks berikut
```python
print(pd.__version__)
```
Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma
